app.py

```diff
@@ -0,0 +1,283 @@
+import os
+import os.path as op
+from flask import Flask
+from flask import request
+from flask_sqlalchemy import SQLAlchemy
+import africastalking
+
+import flask_admin as admin
+from flask_admin.contrib.sqla import ModelView
+
+app = Flask(__name__)
+app.config.from_pyfile('config.py')
+db = SQLAlchemy(app)
+
+username = "sandbox"   
```

Comment:
At some point we'll want to extract these details into separate configs for each environment. @toriejw recently did it for the End Slavery Now Project

Reply:
sounds good - went ahead and added a ticket for this: https://github.com/elixirlabsinc/pangeanetwork/issues/6